cobc/scanner.l

```diff
@@ -1257,11 +1257,76 @@ error_literal (const char *type, const char *literal)
 	cb_error ("%s", err_msg);
 }
 
+/* EBCDIC to ASCII table, used in read_literal; this table was copied from
+   `codegen.c:output_ebcdic_to_ascii_table`.  */
+static const cob_u8_t cob_ebcdic_ascii[256] = {
```

Comment:
To drop an unused warning you likely want to surround that by `#ifndef COB_EBCDIC_MACHINE`.